esp32_ulp/preprocess.py

```diff
@@ -42,12 +82,33 @@ def expand_defines(self, line):
 
         return line
 
+    def expand_rtc_macros(self, line):
+        clean = line.strip()
+        if len(clean) == 0:
+            return line
+
+        macro = clean.split('(', 1)
+        if len(macro) != 2:
+            return line
```

Comment:
add below:
macro_name, macro_args = macro

and then use these instead of macro[0] or splitting again.

Reply:
Very nice! Just did that and looks much cleaner. But it doesn't remove a splittng, because the macro_args need to be split further (one reason why i originally didnt like to use an "args" naming for the 2nd part of the split - because initially it's really the "uncleaned args", but it still looks nicer now. Will push soon)